Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,47 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
+    # install all executable files in "main" directory
+    Dir.foreach("main") do |entry|
+      path = "main/#{entry}"
+      if File.file?(path) && File.executable?(path)
+        bin.install path
+      end
+    end
+  end
+
+  test do
+    # define some variables
+    rate = 16000
+    freq = 100
+    dur = 5
+    bn = 'sine'
```

Comment:
Also let's just do `rate_khz`, `frequency_hz`, `duration_secs` and skip the comment below.